samples/flow/public/index.html

```diff
@@ -10,6 +10,8 @@
 	<link rel='stylesheet' href='/global.css'>
 	<link rel='stylesheet' href='/build/bundle.css'>
 
+  <!-- Expose a GleanJS app id to the code. -->
+  <script id="glean-js" app-id="glean-js-tmp"></script>
```

Comment:
I am not sure we need to add this script tag here. Because we require Glean on Home.svelte, Glean will be added to the bundle.js after building.

Reply:
We require this for the `app-id`